src/sentry/templates/sentry/emails/relocation_failed.html

```diff
@@ -0,0 +1,16 @@
+{% extends "sentry/emails/base.html" %}
+
+{% load i18n %}
+
+{% block main %}
+    <h3>Relocation Failed</h3>
+    <p>Your relocation (id: <i>{{ uuid }}</i>) has failed for the following reason:</p>
+    {%if reason != "" %}
+    <code>
+        {{ reason }}
+    </code>
+    <br>
+    <br>
+    {% endif %}
+    <p>Please contact support for further assistance if necessary.</p>
```

Comment:
Should we have a link to support to make this easier?

Reply:
Done. I've updated the images attached to this PR to reflect this.